.gitlab-ci.yml

```diff
@@ -397,6 +397,7 @@ deploy_demo_nginx:
       -e PWA_2_SUBDOMAIN=b2b
       -e PWA_2_CHANNEL=inSPIRED-inTRONICS_Business-Site
       -e PWA_2_FEATURES=quoting,recently,compare,businessCustomerRegistration,advancedVariationHandling,sentry
+      -e PWA_2_THEME=blue
```

Comment:
I wanna see the blue and I#m not seein it?! https://www.youtube.com/watch?v=eJkbKXTjNrA
C'est bleu? Non estas azure! :rofl: